CONTRIBUTORS.txt

```diff
@@ -72,21 +76,26 @@ needs to be listed here.
 - OleG <emptystamp@gmail.com>
 - Portia <portia-s@users.noreply.github.com>
 - Pushkar N Kulkarni <pushkar.nk@in.ibm.com>
+- Romain Pouclet <rpouclet@apple.com>
 - Rukshani Athapathu <arukshani@users.noreply.github.com>
 - Sebastian Thiebaud <770074+SebastianThiebaud@users.noreply.github.com>
 - Semen Zhydenko <simeon.zhidenko@gmail.com>
 - Sergej Jaskiewicz <jaskiewiczs@icloud.com>
 - Sergo Beruashvili <beruashvili@gmail.com>
 - Shaun Hubbard <shaunhubbard2013@icloud.com>
 - Siemen Sikkema <s.h.sikkema@gmail.com>
+- Simon Evans <si@si.org>
 - Stuart Hinson <stuart.hinson@gmail.com>
 - Tanner <me@tanner.xyz>
 - Tiago Martinho <martinho@thoughtworks.com>
 - Tibor Bödecs <mail.tib@gmail.com>
 - Tim Condon <0xtimc@gmail.com>
 - Tim Kientzle <tkientzle@apple.com>
+- Tom Counsell <tamc@greenonblack.com>
 - Tomer Doron <tomerd@apple.com>
+- Trevör <adtrevor@users.noreply.github.com>
```

Comment:
@adtrevor if you would like me to add a proper email address, please let me know which one.